react/components/Alert/index.test.js

```diff
@@ -5,7 +5,7 @@ import Alert from './index'
 
 // Skipping those tests because getByText is getting the div inside the button and not the button element itself
 // I've tried to fix it, but I couldn't manage to. So I let me fellows future developers in this endeavour
-describe.skip('Alert', () => {
+describe('Alert', () => {
   describe('action button', () => {
     it('should be displayed if props are passed', () => {
       const label = 'Action'
```

Comment:
The tests are not passing are they?

Reply:
before they were not passed, now they are passing.